experiments/CamemBERT/model/CamemBERT.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, model_name: str):
         self.layer3 = AutoModelForTokenClassification.from_pretrained(model_name).classifier
 
     def forward(self, input):
+        x = input.reshape(1, 7, 768)

```

Comment:
Hmmm, I don't see how this works. Shouldn't it be `input =` instead of `x =`? And you already did `example_input = example_input.reshape(1, 7*768)` in line 50 but reshape it back here?